batch/2016-07-01.3.1/swagger/BatchService.json

```diff
@@ -8253,6 +8253,156 @@
         }
       }
     },
+    "/jobs/{jobId}/tasks/{taskId}/reactivate": {
+      "post": {
+        "tags": [
+          "Tasks"
+        ],
+        "operationId": "Task_Reactivate",
+        "summary": "Reactivates the specified task.",
+        "description": "Reactivation makes a task eligible to be retried again up to its maximum retry count. This will fail for tasks that are not completed or that previously completed successfully (with an exit code of 0). Additionally, this will fail if the job has completed (or is terminating or deleting).",
+        "x-ms-request-id": "request-id",
+        "parameters": [
+          {
+            "name": "jobId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The id of the job containing the task."
+          },
+          {
+            "name": "taskId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The id of the task to reactivate."
+          },
+          {
+            "name": "timeout",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "format": "int32",
+            "default": 30,
+            "description": "The maximum time that the server can spend processing the request, in seconds. The default is 30 seconds.",
+            "x-ms-parameter-grouping": {
+              "postfix": "Options"
+            }
+          },
+          {
+            "name": "client-request-id",
+            "in": "header",
+            "required": false,
+            "type": "string",
```

Comment:
No, our request ID I think is not actually required to be a `uuid`  I will doublecheck with our server guys but last time this came up I think they said we should leave it as string.